<launch>
    <include file="$(find realsense2_camera)/launch/rs_camera.launch" >
        <arg name="align_depth" value="true"/>
    </include>

    <node pkg="tf" type="static_transform_publisher" name="tf_camera" args="0 0 1 0 1.571 3.145 map camera_link 100" />

    <include file="$(find find_object_2d)/launch/ros1/find_object_3d.launch" >
        <arg name="rgb_topic" value="/camera/color/image_raw"/>
        <arg name="depth_topic" value="/camera/depth/image_rect_raw"/>
        <arg name="camera_info_topic" value="/camera/color/camera_info"/>
        <arg name="objects_path" value="$(find pose_estimation)/objects"/>
        <arg name="gui" value="false"/>
        <arg name="tf_example" value="true"/>
    </include>

    <!-- <node name="transform_coords" pkg="pose_estimation" type="transform_coords.py" output="screen"/> -->

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find pose_estimation)/rviz/rviz.rviz" />
</launch>

